rules/crowdstrike_rules/crowdstrike_dns_request.py

```diff
@@ -1,17 +1,20 @@
+from panther_base_helpers import get_crowdstrike_field
+
 # baddomain.com is present for testing purposes. Add domains you wish to be alerted on to this list
 DENYLIST = ["baddomain.com"]
 
 
 def rule(event):
-    if event.get("DomainName") in DENYLIST:
+    if get_crowdstrike_field(event, "DomainName") in DENYLIST:
         return True
     return False
 
 
 def title(event):
-    return f"A denylisted domain [{event.get('DomainName')}] was queried by host {event.get('aid')}"
+    # pylint: disable=line-too-long
+    return f"A denylisted domain [{get_crowdstrike_field(event, 'DomainName')}] was queried by host {event.get('aid')}"
 
 
 def dedup(event):
     #  Alert on every individual lookup of a bad domain, per machine
-    return f"{event.get('DomainName')}-{event.get('aid')}"
+    return f"{get_crowdstrike_field(event, 'DomainName')}-{event.get('aid')}"
```

Comment:
`aid` is still populated on the top-level, that's why we don't call `get_crowdstrike_field`?